.LIB_TEMPLATE/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@ledgerhq/TEMPLATE",
+  "version": "1.0.0",
+  "description": "",
+  "keywords": [
+    "Ledger"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LedgerHQ/ledger-live.git"
+  },
+  "bugs": {
+    "url": "https://github.com/LedgerHQ/ledger-live/issues"
+  },
+  "homepage": "https://github.com/LedgerHQ/ledger-live/tree/develop/libs/TEMPLATE",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "lib/index.js",
+  "module": "lib-es/index.js",
+  "types": "lib/index.d.ts",
+  "license": "Apache-2.0",
+  "dependencies": {},
+  "devDependencies": {},
```

Comment:
It might be nice to provide jest as a dep since there is jest config file in the template